### PR TITLE
Zybooks page

### PR DESCRIPTION
**Pull request by adrianlizarraga**
Revised application to display zybooks (subscribed and evaluation) at /zybooks. Addresses issue #2.

### CODE REVIEW

**Comment by adrianlizarraga** [comment]
screen shot 2016-06-08 at 4 26 46 pm

File: app/models/zybook.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    course: null,
+
+    subjects: null,
+
+    public: false,
+
+    isEvaluation: Ember.computed.not('public'),
+
+    init: function() {
+        this.setProperties({
```
Comment (ssirowy) [inline app/models/zybook.js]: There is actual no need to set all of these if they will just be filled in your `fromJSON` method.

**Comment by adrianlizarraga** [comment]
Removed unnecessary init method from zybook model. Once the signin-page branch is merged into master, I'll merge master into this branch.

**Comment by ssirowy** [comment]
@adrianlizarraga can you merge master into this to reduce the PR fingerprint?

**Comment by adrianlizarraga** [comment]
I merged master into this branch.

**Comment by ssirowy** [comment]
@adrianlizarraga code looks good but this PR was set up to be merged into `signin-page` when what you  probably wanted was to merge into `master`.  Github doesnt allow you to change this , so yuo will need to close this PR and then create a new PR.

**Pull request closed without merging**